trunk/SUAVE/Methods/Flight_Dynamics/Static_Stability/Approximations/Tube_Wing/taw_cmalpha.py

```diff
@@ -63,27 +64,26 @@ def taw_cmalpha(geometry,mach,conditions,configuration):
     """
 
     # Unpack inputs
-    Sref  = geometry.reference_area
-    mac   = geometry.wings['main_wing'].chords.mean_aerodynamic
-    c_root= geometry.wings['main_wing'].chords.root
-    taper = geometry.wings['main_wing'].taper
-    c_tip = taper*c_root
-    span  = geometry.wings['main_wing'].spans.projected
-    sweep = geometry.wings['main_wing'].sweeps.quarter_chord
-    C_Law = conditions.lift_curve_slope
-    w_f   = geometry.fuselages['fuselage'].width
-    l_f   = geometry.fuselages['fuselage'].lengths.total
+    Sref   = geometry.reference_area
+    mac    = geometry.wings['main_wing'].chords.mean_aerodynamic
+    c_root = geometry.wings['main_wing'].chords.root
+    taper  = geometry.wings['main_wing'].taper
+    c_tip  = taper*c_root
+    span   = geometry.wings['main_wing'].spans.projected
+    sweep  = geometry.wings['main_wing'].sweeps.quarter_chord
+    C_Law  = conditions.lift_curve_slope
+    alpha  = conditions.aerodynamics.angle_of_attack
+
     x_cg  = configuration.mass_properties.center_of_gravity[0]
-    x_rqc = geometry.wings['main_wing'].origin[0] + 0.5*w_f*np.tan(sweep) + 0.25*c_root*(1 - (w_f/span)*(1-taper))
+
     M     = mach
     
     weights      = conditions.weights.total_mass
```

Comment:
@timdmacdo  take a look at this please. The CG is calculated in simple_sizing within the regression B737 script however I do not think is a part of the tut_mission_B737.py script